### PR TITLE
Correct capitalization for Minecraft Feedback Discord server

### PR DESCRIPTION
**Pull request by dericksonmark**

### CODE REVIEW

**Review by osfanbuff63 — APPROVED** [review]
I feel like I should have caught that, but apparently not 😅

File: assets/messages.yml [hunk 1]
```diff
@@ -323,7 +323,7 @@ messages:
         *Thank you for your report!*
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
-        We're currently not taking bug reports for the Combat Test experiment; it is for testing the combat system only. Please leave any combat-related feedback on the [reddit post|https://www.reddit.com/r/Minecraft/comments/idvujw/here_we_go_again_combat_test_snapshot_8b/] or on the [feedback discord server|https://discord.gg/MXxTqXnuD5].
+        We're currently not taking bug reports for the Combat Test experiment; it is for testing the combat system only. Please leave any combat-related feedback on the [reddit post|https://www.reddit.com/r/Minecraft/comments/idvujw/here_we_go_again_combat_test_snapshot_8b/] or on the [Minecraft Feedback Discord server|https://discord.gg/MXxTqXnuD5].
```
Comment (osfanbuff63) [inline assets/messages.yml:326]: Should we fix the Reddit capitalization too?
```suggestion
        We're currently not taking bug reports for the Combat Test experiment; it is for testing the combat system only. Please leave any combat-related feedback on the [Reddit post|https://www.reddit.com/r/Minecraft/comments/idvujw/here_we_go_again_combat_test_snapshot_8b/] or on the [Minecraft Feedback Discord server|https://discord.gg/MXxTqXnuD5].
```

Comment (dericksonmark) [reply assets/messages.yml]: Might as well :)

**Review by osfanbuff63 — APPROVED** [review]
LGTM

**Pull request merged**